tetris/piece.go

```diff
@@ -296,6 +297,10 @@ func (p *Piece) canMoveLeft(b Board) bool {
 	return true
 }
 
+func (p *Piece) Color() string {
```

Comment:
Why not returning Color?

`func (p *Piece) Color() Color {...}`

Remember, `Color` is a new type, so let's use it.